cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/MonitorConfig.java

```diff
@@ -278,6 +278,20 @@ public final class MonitorConfig {
   public static final String LINEAR_REGRESSION_MODEL_MIN_NUM_CPU_UTIL_BUCKETS_DOC = "The minimum number of full CPU"
       + " utilization buckets required to generate a linear regression model.";
 
+  /**
+   * <code>fetch.metric.samples.retry.count</code>
+   */
+  public static final String FETCH_METRIC_SAMPLES_RETRY_COUNT_CONFIG = "fetch.metric.samples.retry.count";
+  public static final int DEFAULT_FETCH_METRIC_SAMPLES_RETRY_COUNT = 5;
+  public static final String FETCH_METRIC_SAMPLES_RETRY_COUNT_DOC = "The maximum number of retries during the fetch of metric samples.";
```

Comment:
The name and description of this config does not seem accurate. Isn't this config about the maximum number of retries to identify the sample store topic replication factor?